packages/integration-tests/tests/__snapshots__/v1020-multi-project-yarn-auto-convert.test.ts.snap

```diff
@@ -90,15 +90,15 @@ Object {
         "@angular-eslint/component-selector": Array [
           "error",
           Object {
-            "prefix": "customprefix",
+            "prefix": "app",
```

Comment:
These prefixes should not have been changed

Reply:
Idk why it has changed, but after revert everything and do a fresh build/run, it didn't change anymore.